Dockerfile

```
@@ -0,0 +1,15 @@
+FROM golang:1.20-alpine
+
+WORKDIR /app
+
+COPY go.mod ./
+RUN go mod download
+
+COPY ./ ./
+
+WORKDIR /app/cmd/versitygw
+RUN go build -o versitygw
+
+RUN mkdir /tmp/vgw
+
```

Can probably fix it with a multistage build like:
```suggestion
FROM alpine:latest
COPY --from=0 /app/versitygw /app/versitygw
```